pytorch_lightning/trainer/connectors/signal_connector.py

```diff
@@ -0,0 +1,121 @@
+import logging
```

Comment:
Add this to https://github.com/PyTorchLightning/pytorch-lightning/blob/2ec39e275ec10014da6b72af3cab4efad08a096e/pyproject.toml#L63

Reply:
Done !